robotiq_description/CMakeLists.txt

```diff
@@ -10,10 +10,11 @@ find_package(ament_cmake REQUIRED)
 
 install(
   DIRECTORY
-    urdf
+    config
```

Comment:
So, in my opinion, the driver (robotiq_driver) should not know who controls it, that's why I moved the controller config out of it.

On the other hand, the description package (robotiq_description) which contains the robot launch file should know both about the driver and the controller (robotiq_controllers), because you want to use it to start a working robot.

I also think that the urdf in the description package should be of a robot example, not a real robot which may differ a lot from the example. Because the description package knows about the robot and contains the launch file, it seemed natural to add a dependency to rviz too.